client/src/components/pages/RobotOverview/RobotOverview.jsx

```diff
@@ -14,23 +16,40 @@ const { Search } = Input;
  */
 const RobotOverview = () => {
   const [searchValue, setSearchValue] = useState('');
-  const [userId, setUserId] = useState(1);
+  const [userId, setUserId] = useState('80625d115100a2ee8d8e695b');
+  const [robotList, setRobotList] = useState([]);
+
+  /**
+   * @description Fetches Bots for the specified user and will trigger a rerender so that it will be displayed
+   * @param {String} userIuserIdToFetch The userId to fetch Bots for
+   */
+   const retrieveBotList = async (userIuserIdToFetch) => {
+    fetchSSOTsForUser(userIuserIdToFetch)
+      .then((response) => response.json())
+      .then((data) => {
+        setRobotList(data);
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+  };
 
   /**
    * @description Equivalent to ComponentDidMount in class based components
    */
-  useEffect(() => {
-    initSessionStorage('CurrentUserId', 1);
-    setUserId(sessionStorage.getItem('CurrentUserId'));
+  useEffect(async () => {
+    initSessionStorage('CurrentUserId', '80625d115100a2ee8d8e695b');
+    retrieveBotList(userId);
```

Comment:
Using async directly in the useEffect function isn't allowed. It also shouldn't be necessary here. The code should work without that async